frontend/src/Components/Plantdetailcard/PlantDetailCard.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import monsteraImage from '../../assets/plant-images/3.jpg'
+
+export default function PlantDetailCard({ species, info, className }) {
+  PlantDetailCard.propTypes = {
+    species: PropTypes.string.isRequired,
+    info: PropTypes.string.isRequired,
+    className: PropTypes.string,
+  }
+  return (
+    <CardWrapper className={className}>
+      <ImageContainer src={monsteraImage}></ImageContainer>
```

Comment:
You don't embed anything in the `ImageContainer` so you don't need your the closing tag. Maybe something like `ImageFrame` would be more appropriate.